lib/organization/role.ex

```diff
@@ -0,0 +1,43 @@
+defmodule GlassFactoryApi.Organization.Role do
+  @moduledoc """
+  Defines a Role of an Organization.
+  """
+
+  @type t() :: %__MODULE__{
+          id: String.t(),
+          name: String.t(),
+          description: String.t(),
+          department_id: String.t(),
+          is_used: boolean(),
+          rate: float(),
```

Comment:
If this `rate` field is related to currency somehow, like a modifier or something, we should avoid floats too. Decimal is a safer choice for these kind of calculations.

Reply:
Rate is the price of a working hour of this Role, I'm changing to  string as well